lib/emision_conversion.rb

```diff
@@ -0,0 +1,16 @@
+class EmissionConversion
+  # Multiply carbon value in kg by these to convert to the respective unit
```

Comment:
Thanks for adding these :)

Only issue is that this isn't the easiest comment to understand! :P it's fine while it's just us though

Reply:
I may still change it since it kinda defeats the point if my comment explaining it ALSO needs its own explanation.

Reply:
Also just realised that "emission" isn't spelled correctly in this filename, which surprises me since I thought Rails was pretty strict on that sort of thing